### PR TITLE
Replace `serve-static` with `sirv`

### PR DESCRIPTION
**Pull request by lukeed**
I would also suggest setting `maxAge` and `immutable` values (for production only), but leaving it out for the template may have been intentional.

_Closes #63_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "compression": "^1.7.1",
     "polka": "^0.3.4",
     "sapper": "^0.12.0",
-    "serve-static": "^1.13.1"
+    "serve-static": "^1.13.1",
```
Comment (Conduitry) [inline package.json]: serve-static should be completely unnecessary now, no?

Comment (lukeed) [reply package.json]: whoops, not sure what happened here

**Comment by Rich-Harris** [comment]
thanks! here's some numbers you might like:

## with serve-static

```bash
➜  ~ wrk -t12 -c400 -d30s http://localhost:3000
Running 30s test @ http://localhost:3000
  12 threads and 400 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency    35.65ms    5.21ms 121.19ms   87.52%
    Req/Sec   560.25    219.43     1.02k    54.71%
  200985 requests in 30.07s, 511.58MB read
  Socket errors: connect 157, read 130, write 0, timeout 0
Requests/sec:   6684.08
Transfer/sec:     17.01MB
```

## with sirv

```bash
➜  ~ wrk -t12 -c400 -d30s http://localhost:3000
Running 30s test @ http://localhost:3000
  12 threads and 400 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency    31.25ms    4.91ms 156.48ms   87.91%
    Req/Sec   639.29    216.35     1.18k    70.21%
  229299 requests in 30.08s, 583.65MB read
  Socket errors: connect 157, read 132, write 0, timeout 0
Requests/sec:   7624.14
Transfer/sec:     19.41MB
```

**Comment by lukeed** [comment]
💃 Avoiding file system ftw~!

**Pull request merged**